src/js/components/EmptyStateView.jsx

```diff
@@ -37,6 +51,6 @@ export class EmptyStateView extends Component {
 
 EmptyStateView.propTypes = {
     children: PropTypes.node,
-    iconName: PropTypes.oneOf(['branch','goat','shoes']),
+    iconName: PropTypes.string,
```

Comment:
fair enough. My only remaining objection is the word "goat" is no longer found in the codebase.